src/DependencyInjection/SimpleThingsEntityAuditExtension.php

```diff
@@ -65,8 +65,8 @@ private function fixParametersFromDoctrineEventSubscriberTag(ContainerBuilder $c
     {
         foreach ($definitionNames as $definitionName) {
             $definition = $container->getDefinition($definitionName);
-            $tags = $definition->getTag('doctrine.event_subscriber');
-            $definition->clearTag('doctrine.event_subscriber');
+            $tags = $definition->getTag('doctrine.event_listener');
```

Comment:
The method `fixParametersFromDoctrineEventSubscriberTag` can be renamed
`fixParametersFromDoctrineEventListenerTag`